python/src/iceberg/utils/datetime.py

```diff
@@ -63,3 +65,27 @@ def timestamptz_to_micros(timestamptz_str: str) -> int:
     if ISO_TIMESTAMPTZ.fullmatch(timestamptz_str):
         return datetime_to_micros(datetime.fromisoformat(timestamptz_str))
     raise ValueError(f"Invalid timestamp with zone: {timestamptz_str} (must be ISO-8601)")
+
+
+def to_human_day(day_ordinal: int) -> str:
+    """Converts a DateType value to human string"""
+    to_time = EPOCH_TIMESTAMP + timedelta(days=day_ordinal)
```

Comment:
This should not use `EPOCH_TIMESTAMP` because that includes time. Instead, use `EPOCH_DATE`.

Reply:
Fixed.